drivers/src/nrfx_nvmc.c

```diff
@@ -415,6 +460,19 @@ void nrfx_nvmc_words_write(uint32_t addr, void const * src, uint32_t num_words)
     nvmc_readonly_mode_set();
 }
 
+#if defined(UICR_OTP_UPPER_Msk) // To check whether OTP region is present
```

Comment:
It's not a good idea to cut out the implementation of this function while it is still declared in the header file.
But I don't think there is a point in using this `#if` at all. The function in its current form can be successfully compiled even if the OTP region is not present. And if the function appears to be unused, the linker should be able to discard its code.